### PR TITLE
Prepare release 0.8.5

### PR DESCRIPTION
**Pull request by maxlambrecht**
Bump library version to 0.8.5
Update CHANGELOG and READMEs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+- Updated Gradle to version 8.5 (#201)
+- Various enhancements in `java-spiffe-helper` (#199)
+
+### Fixed
+
+- Addressed a Fat Jar Assembly issue. (#198)
+
+### Dependencies updates
+
+- Bump `grpcVersion` from 1.54.0 to 1.61.1 (#202)
```
Comment (rturner3) [inline CHANGELOG.md:20]: nit: can we elaborate on the exact packages that got bumped?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+- Bump `commons-validator:commons-validtor` from 1.7.0 to 1.8.0 (#197)
```
Comment (rturner3) [inline CHANGELOG.md:21]:
```suggestion
- Bump `commons-validator:commons-validator` from 1.7. to 1.8.0 (#197)
```

- typo in package name
- previous version didn't have a patch component - just `1.7` vs. `1.7.0`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+- Bump `commons-cli:commons-cli` from 1.5.0 to 1.6.0 (#197)
```
Comment (rturner3) [inline CHANGELOG.md:22]:
```suggestion
- Bump `commons-cli:commons-cli` from 1.5.0 to 1.6.0 (#196)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+- Bump `com.google.protobuf` from 3.21.12 to 3.25.2 (#193)
```
Comment (rturner3) [inline CHANGELOG.md:23]:
```suggestion
- Bump `com.google.protobuf:protoc` from 3.21.12 to 3.25.2 (#193)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+- Bump `io.netty:netty-transport-native-kqueue` from 4.1.91.Final to 4.1.106.Final (#192)
+- Bump `org.apache.commons:commons-lang3` from 3.13.0 to 3.14.0 (#189)
```
Comment (rturner3) [inline CHANGELOG.md:25]:
```suggestion
- Bump `org.apache.commons:commons-lang3` from 3.12.0 to 3.14.0 (#189)
```

https://github.com/spiffe/java-spiffe/blob/537e7f7a7b40fb6d2e3323e2bf2a64be1f63b35c/build.gradle#L102

https://github.com/spiffe/java-spiffe/blob/537e7f7a7b40fb6d2e3323e2bf2a64be1f63b35c/java-spiffe-core/build.gradle#L98

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,33 @@
...
+### Changed
+
+- Updated Gradle to version 8.5 (#201)
+- Various enhancements in `java-spiffe-helper` (#199)
+
+### Fixed
+
+- Addressed a Fat Jar Assembly issue. (#198)
+
+### Dependencies updates
```
Comment (rturner3) [inline CHANGELOG.md:18]:
```suggestion
### Dependency updates
```

**Comment by rturner3** [comment]
I think the accepted suggestions in the GitHub UI are causing DCO to fail. Maybe squash them locally and force push?

File: CHANGELOG.md (hunk 1 above)
Comment (maxlambrecht) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+- Various enhancements in `java-spiffe-helper` (#199)
+
+### Fixed
+
+- Addressed a Fat Jar Assembly issue. (#198)
+
+### Dependency updates
+
+- Bump `io.grpc:grpc-protobuf` and  `io.grpc:grpc-stub` from 1.54.0 to 1.61.1 (#202)
+- Bump `commons-validator:commons-validator` from 1.7. to 1.8.0 (#197)
```
Comment (rturner3) [inline CHANGELOG.md:21]:
```suggestion
- Bump `commons-validator:commons-validator` from 1.7 to 1.8.0 (#197)
```

nit: stray `.`

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,33 @@
...
+- Updated Gradle to version 8.5 (#201)
+- Various enhancements in `java-spiffe-helper` (#199)
+
+### Fixed
+
+- Addressed a Fat Jar Assembly issue. (#198)
+
+### Dependency updates
+
+- Bump `io.grpc:grpc-protobuf` and  `io.grpc:grpc-stub` from 1.54.0 to 1.61.1 (#202)
```
Comment (rturner3) [inline CHANGELOG.md:20]:
```suggestion
- Bump `io.grpc:grpc-protobuf` and `io.grpc:grpc-stub` from 1.54.0 to 1.61.1 (#202)
```
nit: stray space

**Review by rturner3 — APPROVED** [review]
Noticed just a couple very small formatting blips, but otherwise LGTM

**Pull request merged**